src/valid/mod.rs

```diff
@@ -58,6 +58,7 @@ bitflags::bitflags! {
         const PUSH_CONSTANT = 0x1;
         /// Float values with width = 8.
         const FLOAT64 = 0x2;
+        const PRIMITIVE_ID = 0x4;
```

Comment:
similarly, let's call it `PRIMITIVE_INDEX`

Reply:
Should we maybe call it `GEOMETRY` to reflect the generic shader capability?

Reply:
No, we aren't going to support geometry shaders any time soon. From the point of view of Naga, this should be a capability unrelated to geometry shaders. It's just the Naga client's responsibility to check on Vulkan that geometry is allowed.